communication/slack-config/channels.yaml

```diff
@@ -25,6 +25,12 @@ channels:
   - name: cka-exam-prep
   - name: ckad-exam-prep
   - name: client-go-docs
+  - name: cloud-provider-aws
```

Comment:
@jeefy @Katharine Would it be better to rename these?

Reply:
Absolutely. You can rename them by changing the name attribute, and adding the "id" attribute (see `ug-big-data` for example)

I forget if those identifiers are accessible by non-admins or not. Worst case, create an issue with what the channels should be renamed from/to and we can cut a PR pretty quick after this gets merged.

Reply:
agreed, was wondering actually if there's a way to rename from here or will it require manual intervention from a slack admin?

Reply:
Non-admins can indeed access these IDs.

Reply:
It's somewhat of a convoluted way to get them. The easiest way that I know of is to view them in the slack web client. 
It'll be the trailing part of the url e.g. `ug-big-data`, formally `sig-big-data` is `C0ELB338T` and its web url is https://app.slack.com/client/T09NY5SBT/C0ELB338T

Reply:
For sake of convenience:

sig-aws: `C0LRMHZ1T`
sig-azure: `C5HJXTT9Q`
sig-gcp: `C6WB33KNJ`
sig-ibmcloud: `C90BWDWCE`
sig-openstack: `C0LSA3T7C`
sig-vmware: `C9PGCDKV5`

Reply:
Updated to specify the channel IDs, PTAL :)

Reply:
[Tempelis says](https://prow.k8s.io/view/gcs/kubernetes-jenkins/pr-logs/pull/community/3895/pull-community-tempelis-check/1151209996575313921):

```
In dry run mode so taking no action, but this is what we would've done:
Step 1: Rename channel C0LRMHZ1T from sig-aws to cloud-provider-aws.
Step 2: Rename channel C5HJXTT9Q from sig-azure to cloud-provider-azure.
Step 3: Rename channel C6WB33KNJ from sig-gcp to cloud-provider-gcp.
Step 4: Rename channel C90BWDWCE from sig-ibmcloud to cloud-provider-ibmcloud.
Step 5: Rename channel C0LSA3T7C from sig-openstack to cloud-provider-openstack.
Step 6: Rename channel C9PGCDKV5 from sig-vmware to cloud-provider-vmware.
```

Reply:
Fixed now, thanks for catching that